tensorbay/opendataset/BDD100K/loader.py

```diff
@@ -279,6 +363,69 @@ def _merge_label(source_label_contents: List[List[Dict[str, Any]]]) -> Dict[str,
     return label_contents
 
 
+def _get_masks(
+    filename: str,
+    original_mask_paths: Dict[str, str],
+    label: Label,
+    single_channel_mask_paths: Dict[str, str],
+) -> None:
+    label.semantic_mask = SemanticMask(os.path.join(original_mask_paths["sem"], f"{filename}.png"))
+    label.instance_mask = _get_mask(
+        filename, original_mask_paths["ins"], single_channel_mask_paths["ins"], "ins"
+    )
+    label.panoptic_mask = _get_mask(
+        filename, original_mask_paths["pan"], single_channel_mask_paths["pan"], "pan"
+    )
+
+
+def _get_mask(filename: str, original_mask_path: str, mask_directory: str, seg_type: str) -> Any:
+    mask_path = os.path.join(mask_directory, f"{filename}.png")
+    if not os.path.exists(mask_path):
+        mask = np.array(Image.open(os.path.join(original_mask_path, f"{filename}.png")))
+        mask_info = _get_mask_info(mask, seg_type)
+        Image.fromarray(mask[:, :, -1]).save(mask_path)
+        with open(os.path.join(mask_directory, f"{filename}.json"), "w") as fp:
+            json.dump(mask_info, fp)
+        if seg_type == "pan":
+            seg_mask: Any = PanopticMask(mask_path)
```

Comment:
Why `Any`?

Reply:
`seg_mask` may be `PanopticMask` or `InstanceMask`, without `Any` the below `InstanceMask` code will raise assignment error

Reply:
you should not use one variable for different types in most cases.
can use different variables for `PanopitcMask` and `InstanceMask` here?